NakedJavaScript/mensa-projekt
# [cookies] Added cookies

# Added Cookies to the Site

## How to test?

* Load the site
* A cookie warning should show up at the bottom of the page
* If it doesn't work try to delete the cookies for your browser
